template/src/extension.ts

```diff
@@ -1,66 +1,39 @@
-import { Handler, HandlerArgs, registerFeedbackButton } from '../../package/lib'
+import { Handler, initLSIF } from '../../package/lib'
 import * as sourcegraph from 'sourcegraph'
 import { languageSpecs } from '../../languages'
-import { documentSelector } from '../../package/lib/handler'
 
 const DUMMY_CTX = { subscriptions: { add: (_unsubscribable: any) => void 0 } }
 
 export function activate(ctx: sourcegraph.ExtensionContext = DUMMY_CTX): void {
     // This is set to an individual language ID by the generator script.
     const languageID = 'all'
 
-    if (languageID === 'all') {
-        for (const languageSpec of languageSpecs) {
-            activateWithArgs(ctx, { ...languageSpec.handlerArgs, sourcegraph })
+    for (const languageSpec of languageID === 'all'
+        ? languageSpecs
+        : [languageSpecs.find(l => l.handlerArgs.languageID === languageID)!]) {
+        if (sourcegraph.configuration.get().get('codeIntel.lsif')) {
+            initLSIF()
```

Comment:
Doesn't `initLSIF()` return an object? What's the reason it is not used here?

Reply:
I think this is old code.